RM_SUI/RM_SUI/Extension+UIImage.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Extension+UIImage.swift
+//  RM_SUI
+//
+//  Created by Дмитрий Супрун on 21.01.23.
+//
+
+import UIKit
```

Comment:
```suggestion
import UIKit
```
а зачем нам понадолюился UIKit, почему не SUI?